includes/Hooks.php

```diff
@@ -132,18 +132,20 @@ public static function onBeforeCreateEchoEvent(
 		&$notifications, &$notificationCategories, &$icons
 	) {
 		$notificationCategories['wiki-creation'] = [
-			'priority' => 3,
+			'priority' => 1,
 			'tooltip' => 'echo-pref-tooltip-wiki-creation',
+			'no-dismiss' => [ 'all' ]
 		];
 
 		$notificationCategories['request-declined'] = [
-			'priority' => 3,
+			'priority' => 2,
 			'tooltip' => 'echo-pref-tooltip-wiki-request-declined'
 		];
 
 		$notificationCategories['request-moredetails'] = [
-			'priority' => 3,
-			'tooltip' => 'echo-pref-tooltip-wiki-request-moredetails'
```

Comment:
When using `'no-dismiss' => [ 'all' ],` we don't need tooltips, you can remove this here and from i18n if you want.